e2e_tests/model.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: example-sklearn-isvc
+  annotations:
+    serving.kserve.io/deploymentMode: ModelMesh
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      runtime: mlserver-0.x
+      storageUri: "https://github.com/ruivieira/kserve-trustyai-explainer/blob/predictor-model/predictor/model.joblib?raw=true"
```

Comment:
We should move this into the repo or in general use a [permanent link](https://github.com/ruivieira/kserve-trustyai-explainer/blob/55ef1fa76379ddad543dac2d130b5a4644449f02/predictor/model.joblib?raw=true) to be safer